packages/grafana-ui/src/components/BigValue/BigValue.story.tsx

```diff
@@ -32,11 +14,84 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disabled: true,
+    },
+  },
+  argTypes: {
+    width: { control: { type: 'range', min: 200, max: 800 } },
+    height: { control: { type: 'range', min: 200, max: 800 } },
+    colorMode: { control: { type: 'select', options: [BigValueColorMode.Value, BigValueColorMode.Background] } },
+    graphMode: { control: { type: 'select', options: [BigValueGraphMode.Area, BigValueGraphMode.None] } },
+    justifyMode: { control: { type: 'select', options: [BigValueJustifyMode.Auto, BigValueJustifyMode.Center] } },
+    textMode: {
+      control: {
+        type: 'select',
+        options: [BigValueTextMode.Auto, BigValueTextMode.Name, BigValueTextMode.ValueAndName, BigValueTextMode.None],
+      },
+    },
+    color: { control: 'color' },
+    sparkline: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
Maybe you should extract disabled/hidden controls to a shared const so that all stories could reuse it. WDYT?
```ts
export const NOOP_CONTROL = { table: { disable: true } };
```
```suggestion
    sparkline: NOOP_CONTROL,
```

Reply:
I think it's a great idea 👍

Reply:
I think Torkel's discussion today about how we can maintain this moving forward is important too. I mean, when props change in the component it should be minimal effort to keep these disabled props in sync.